src/sentry/testutils/helpers/task_runner.py

```diff
@@ -41,8 +41,8 @@ def BurstTaskRunner():
 
     job_queue = []
 
-    def apply_async(self, args=(), kwargs=(), countdown=None, queue=None):
-        job_queue.append((self, args, kwargs))
+    def apply_async(self, args=(), kwargs=None, countdown=None, queue=None):
```

Comment:
What's the reason behind this change?

Reply:
On line 54, we try to call `**kwargs`, which ofc won't work on a tuple. I think whoever originally wrote this code never used it with a non-default value. Specifically, I suspect that all of the code tested via this runner used `.delay()` to schedule tasks, which is a convenience wrapper around `.apply_async()` and automatically passes `{}` to its `kwargs` param. So in practice, all prior invocations ignored the incorrect default value of `()` and used `{}` under the hood.

Now that we are using this with a test where we call `.apply_async()` directly, so in cases where we don't explicitly set `kwargs`, the default actually gets invoked. So we need to make sure that in cases where no `kwargs` is passed, we use an empty dict.